archunit/src/test/java/com/tngtech/archunit/testutil/Assertions.java

```diff
@@ -134,8 +132,8 @@ public static JavaFieldsAssertion assertThatFields(Iterable<JavaField> fields) {
         return new JavaFieldsAssertion(fields);
     }
 
-    public static JavaClassesAssertion assertThat(JavaClass[] javaClasses) {
-        return new JavaClassesAssertion(javaClasses);
+    public static JavaTypesAssertion assertThat(JavaType[] javaClasses) {
```

Comment:
This could use the parameter  `javaTypes`.

Reply:
True, I saw the same for `assertThatTypes(javaClasses)` -> fixed it :+1: